data/volunteer.md

```diff
@@ -40,3 +40,7 @@ space for events, or catering.
 
 Sponsors will get prominent display on event pages as well as the option to
 distribute swag at events.
+
+#### Interested in hosting your own RustBridge chapter?
+
+See the resources section for awesome tips, outlines and guides!
```

Comment:
this could be a link, i feel

Reply:
Make 
#### Interested in hosting your own RustBridge chapter? 

link directly to the in-a-box repo?

Reply:
@technetos good idea!